src/ResourceManager/Resources/Commands.Resources.Test/ScenarioTests/LocationTests.ps1

```diff
@@ -0,0 +1,30 @@
+﻿# ----------------------------------------------------------------------------------
+#
+# Copyright Microsoft Corporation
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ----------------------------------------------------------------------------------
+
+<#
+    .SYNOPSIS
+    Tests getting azure locations.
+#>
+function Test-AzureLocation
+{
+    $providerLocations = Get-AzureRmLocation
```

Comment:
@TianoMS This test can only be executed against the production AzureCloud endpoint.  It will fail if executed against a test endpoint or the AzureChinaCloud, AzureUSGovernment, or AzureGermanCloud endpoints.  Please simply test that there is a non-empty list of locations and that each of the properties for each location have the appropriate properties.